### DISCUSSION TITLE
My Copilot preview ended, before I even knew it started?!

### DISCUSSION THREAD

**Discussion by Nikorasu** [community/community#30826] [Copilot Conversations]
I just got an email that my Copilot preview has ended.. But I didn't even realize it had actually started yet! Is there anything that can be done? I was not ready yet! Never got to try it even once..

I'm using Linux, but I have no idea how to use Nvim (how can anyone use that?).. So I'm forced to wait until they support something else I can use in Linux, like the Atom editor.. I really didn't want my preview to start yet..

**Comment by iamcjwilliams** [comment]
@Nikorasu Can you provide screenshots of your email correspondence? Also, The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted :)
